### ISSUE TITLE
Keyboard Accessibility of Room/Profile Avatar Upload

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#11287]
Currently it is not tab indexable and thus not keyboard accessible.
Not as simple as making it into an AccessibleButton as a Space keypress cannot seemingly be turned into a click event on the file uploader
